lib/matplotlib/cbook.py

```diff
@@ -2701,3 +2702,15 @@ def __exit__(self, exc_type, exc_value, traceback):
                     os.rmdir(path)
                 except OSError:
                     pass
+
+
+def is_matrix(obj):
+    '''
+    This is a test for whether the input is a matrix.
```

Comment:
Can you please use triple double quotes `"""`: this is the convention for docstring, not triple single quotes.

The docstring is also very unclear on what this function does.